dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/FeatureManager.java

```diff
@@ -1712,74 +1719,44 @@ boolean reportErrors(Result result, Collection<String> restrictedAccessAttempts,
+                                 getEeCompatiblePlatform(conflictRecord1.conflict, ignoreVersion));
                     } else {
                         // One conflicting feature supports "Jakarta EE X", the other "Java EE X"
-                        Tr.error(tc, "UPDATE_CONFLICT_INCOMPATIBLE_EE_FEATURES_DIFFERENT_PLATFORM_ERROR", getPreferredEePlatform(eeConflict1),
-                                 getPreferredEePlatform(eeConflict2), getFeatureName(eeConflict1), getFeatureName(eeConflict2), getFeatureName(configured1),
-                                 getFeatureName(configured2));
+                        Tr.error(tc, "UPDATE_CONFLICT_INCOMPATIBLE_EE_FEATURES_DIFFERENT_PLATFORM_ERROR", getPreferredEePlatform(conflictRecord1.compatibleConflict),
+                                 getPreferredEePlatform(conflictRecord2.compatibleConflict), getFeatureName(conflictRecord1.compatibleConflict),
+                                 getFeatureName(conflictRecord2.compatibleConflict), getFeatureName(conflictRecord1.configured),
+                                 getFeatureName(conflictRecord2.configured));
                         // Remove the conflicting features (not necessarily the configured features)
-                        result.getResolvedFeatures().remove(getFeatureName(eeConflict1));
-                        result.getResolvedFeatures().remove(getFeatureName(eeConflict2));
+                        result.getResolvedFeatures().remove(getFeatureName(conflictRecord1.compatibleConflict));
+                        result.getResolvedFeatures().remove(getFeatureName(conflictRecord2.compatibleConflict));
                     }
                 } else {
-                    Tr.error(tc, "UPDATE_CONFLICT_FEATURE_ERROR", getFeatureName(conflict1), getFeatureName(conflict2), getFeatureName(configured1), getFeatureName(configured2));
+                    Tr.error(tc, "UPDATE_CONFLICT_FEATURE_ERROR", getFeatureName(conflictRecord1.conflict), getFeatureName(conflictRecord2.conflict),
+                             getFeatureName(conflictRecord1.configured), getFeatureName(conflictRecord2.configured));
                 }
-                reportedConfigured.add(new SimpleImmutableEntry<String, String>(configured1, configured2));
+                reportedConfigured.add(new SimpleImmutableEntry<String, String>(conflictRecord1.configured, conflictRecord2.configured));
             }
-            String conflictMsg = "Unable to load conflicting versions of features \"" + conflict1 + "\" and \"" + conflict2 +
-                                 "\".  The feature dependency chains that led to the conflict are: " + chain1 + " and " + chain2;
+            String conflictMsg = "Unable to load conflicting versions of features \"" + conflictRecord2.conflict + "\" and \"" + conflictRecord2.conflict +
```

Comment:
Change 
`"Unable to load conflicting versions of features \"" + conflictRecord2.conflict + ...` 
to 
`"Unable to load conflicting versions of features \"" + conflictRecord1.conflict + ...`

Reply:
Good catch, I'll fix this up along with adding FAT tests.

Reply:
This is resolved now.